website/docs/api/span.md

```diff
@@ -501,11 +501,6 @@ If the span happens to cross sentence boundaries, only the first sentence will
 be returned. If it is required that the sentence always includes the full span,
 the result can be adjusted as such:
 
-```python
-sent = span.sent
-sent = doc[sent.start : max(sent.end, span.end)]
-```
-
```

Comment:
Why did you remove this part from the `span.sent` documentation?

Reply:
@svlandeg Devinitely not on purpose :) For some reason, my PyCharm doesn't like editing .md files. Sorry about that, I will make sure I don't screw up the documentation any more :)